src/utils/optimizeCallback.js

```diff
@@ -42,7 +55,21 @@ function _findSingleSignatureWithArity (callback, arity) {
   }
 }
 
-function _findNumberOfArguments (callback, value, index, array) {
+function _findIfCallbackIsUnary (callback) {
+  const callbackStr = callback.toString()
+  const firstParenthesisIndex = callbackStr.indexOf('(')
+  const firstClosingParenthesisIndex = callbackStr.indexOf(')', firstParenthesisIndex)
+  const paramsStr = callbackStr.slice(firstParenthesisIndex + 1, firstClosingParenthesisIndex)
+
+  if (callback.length === 1) {
+    if (/arguments/.test(callbackStr)) return false
```

Comment:
Can you add two comments explaining what the two regexps do? I also think the second regexp only has to check for three dots, not perse whether followed by a word.